### PR TITLE
Add all the appropriate templates

### LINKED ISSUE #63: Add code of conduct, issue and PR templates, etc.
@lee-dohm: do we have these in a form that would be easy for me to import here?

### PR DESCRIPTION
**Pull request by lee-dohm**
Fixes #63

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+**Expected behavior:** [What you expect to happen]
```
Comment (Arcanemagus) [inline ISSUE_TEMPLATE.md]: I'd suggest moving the `[ ... ]` onto the next line, if the template actually gets filled in then it is duplicate information, and easier to remove when it's on a line of it's own.

Note that `### Description` is already setup like this, and `atom/github` uses this style 😉.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+**Actual behavior:** [What actually happens]
```
Comment (Arcanemagus) [inline ISSUE_TEMPLATE.md]: Same here.

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+**Reproduces how often:** [What percentage of the time does it reproduce?]
```
Comment (Arcanemagus) [inline ISSUE_TEMPLATE.md]: And here 😉.

**Comment by smashwilson** [comment]
:+1: Thanks, @lee-dohm!

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (smashwilson) [reply ISSUE_TEMPLATE.md]: Change: made. :smile:

**Comment by lee-dohm** [comment]
@Arcanemagus You make some good points and I'm going to update the master templates and blast them out so that all repos can benefit :+1:

**Pull request merged**